entity-framework/core/querying/user-defined-function-mapping.md

```diff
@@ -89,6 +89,52 @@ SELECT 100 * (ABS(CAST([p].[BlogId] AS float) - 3) / ((CAST([p].[BlogId] AS floa
 FROM [Posts] AS [p]
 ```
 
+## Configuring nullability of user-defined function based on it's arguments
+
+If the user-defined function can only return `null` when one or more of it's arguments are `null` EFCore provides way to specify that, which results in more performant SQL. It can be done by adding a `PropagatesNullability()` call to the relevant function parameters model configuration.
```

Comment:
```suggestion
If the user-defined function can only return `null` when one or more of its arguments are `null`, EFCore provides way to specify that, resulting in more performant SQL. It can be done by adding a `PropagatesNullability()` call to the relevant function parameters model configuration.
```